src/main/java/io/confluent/kafkarest/KafkaRestContextImpl.java

```diff
@@ -21,7 +21,7 @@
 /**
  * Shared, global state for the REST proxy server, including configuration and connection pools.
  */
-public class Context {
+public class KafkaRestContextImpl implements KafkaRestContext {
```

Comment:
perhaps `DefaultKafkaRestContext` or `BasicKafkaRestContext` or `SimpleKafkaRestContext`? i think maybe the last one is my favorite of the lot.